bot/cogs/utility/nickrequest.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, bot: Bot):
     @commands.command(name="nick")
     # @commands.cooldown(1, command_timeout, commands.BucketType.user)
     async def nick(self, ctx: commands.Context, *, nick):
-        acc_channel = self.bot.get_channel(accept_channel_id)
+        acc_channel = self.bot.get_channel(754818272528040026)
```

Comment:
Why is this necessary, it's better to have this in a config file not hardcoded

Reply:
Resolved (: